### PR TITLE
Migrate pr-kops-aws from Jenkins to Prow

### PR DESCRIPTION
**Pull request by krzyzacy**
- seems k8s 1.7 can be built with bazel 0.7 - https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/52755/pull-kubernetes-e2e-kops-aws-prow/21/, and I clustered all branch jobs here in one entry
- master is passing on https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/46662/pull-kubernetes-e2e-kops-aws-prow/23/
- probably need more prow nodes, waiting for a prow quota bump
/hold
/assign @BenTheElder

### CODE REVIEW

**Comment by BenTheElder** [comment]
> seems k8s 1.7 can be built with bazel 0.7

we should probably do 0.8, 0.7 had some batch issues we haven't found yet since switching k/t-i over.

**Comment by BenTheElder** [comment]
discussed offline, this actually used 0.6.1 which is fine

File: jobs/config.json [hunk 1]
```diff
@@ -10560,6 +10560,9 @@
     ]
   },
   "pull-kubernetes-e2e-kops-aws": {
+    "_comment1": "NOTE: THESE ARE THE JENKINS ARGS / COMMON ARGS (!)",
```
Comment (BenTheElder) [inline jobs/config.json]: :+1:

**Comment by krzyzacy** [comment]
(turning up more ni-highmem-8 nodes :shipit: ) will merge tomorrow

**Comment by krzyzacy** [comment]
@BenTheElder lgtm? node upgrade is done :-)

File: jobs/config.json [hunk 2]
```diff
@@ -10564,27 +10567,6 @@
       "sig-aws"
     ]
   },
-  "pull-kubernetes-e2e-kops-aws-prow": {
```
Comment (BenTheElder) [inline jobs/config.json]: you don't want to keep the canary job?

File: prow/config.yaml [hunk 3]
```diff
@@ -1459,89 +1459,22 @@ presubmits:
     run_if_changed: '^(cluster/gce|cluster/addons).*$'
   - name: pull-kubernetes-e2e-kops-aws
     agent: jenkins
+    branches:
+    - release-1.6
```
Comment (BenTheElder) [inline prow/config.yaml]: :+1: we should make an issue about cleaning these up post 1.9 maybe

**Comment by BenTheElder** [comment]
LGTM comments
/lgtm
/hold

File: jobs/config.json (hunk 2 above)
Comment (krzyzacy) [reply jobs/config.json]: (make canary -> canary pass -> flip canary to prod, I think we don't need the canary anymore)

**Comment by krzyzacy** [comment]
:crossed_fingers: 
/hold cancel

**Comment by krzyzacy** [comment]
(patched up tot id, the recent runs are pretty green)

**Pull request merged**